### PR TITLE
Adds a "salty cream" donut.

### PR DESCRIPTION
**Pull request by Coolgat3**
[Changelogs]: 

:cl: Coolgat3
add: Added the code for the semen donut and made it craftable
imageadd: Added the donut sprites
/:cl:

[why]:
Because I'm a degenerate and people should forcefeed this to their friends.
Also because why not, honestly.

Here is the preview for the sprite as requested by Ragolution:
https://i.imgur.com/pSpohvx.png

The inhand spirtes are pretty much the sprinkled donut, but the pink is replaced with white. That's all.

### CODE REVIEW

**Comment by Ragolution** [comment]
Can you add a preview of the new icon?

File: code/modules/food_and_drinks/food/snacks_pastry.dm [hunk 1]
```diff
@@ -89,13 +89,24 @@
...
+
 /obj/item/reagent_containers/food/snacks/donut/meat
 	bonus_reagents = list("ketchup" = 1)
 	list_reagents = list("nutriment" = 3, "ketchup" = 2)
 	tastes = list("meat" = 1)
 	foodtype = JUNKFOOD | MEAT | GROSS | FRIED
-	
+
+/obj/item/reagent_containers/food/snacks/donut/semen
+	name = "semen donut"
```
Comment (Ragolution) [inline code/modules/food_and_drinks/food/snacks_pastry.dm]:
```suggestion
	name = "\"cream\" donut"
```
Honestly 'semen donut' seems too on-the-nose.

File: code/modules/food_and_drinks/food/snacks_pastry.dm [hunk 2]
```diff
@@ -89,13 +89,24 @@
...
+	desc = "It's rather sticky."
```
Comment (Ragolution) [inline code/modules/food_and_drinks/food/snacks_pastry.dm]:
```suggestion
	desc = "That cream looks a little runny..."
```

**Comment by Coolgat3** [comment]
Editted the PR description, adding a picture of the sprite as requested by Ragolution

**Review by silicons — APPROVED** [review]

**Pull request merged**